From 16896d05a6a2a91fd53cf6af56ce5e79f6abb500 Mon Sep 17 00:00:00 2001
From: Corentin Rossignon <corentin.rossignon@gmail.com>
Date: Thu, 4 Jan 2018 08:34:01 +0100
Subject: [PATCH 107/123] glamor: Specify GLSL version for xv shader

SAMPLE is a GLSL keyword in newer OpenGL version.
This fix issue with gnome-shell and playing video using xv

Signed-off-by: Corentin Rossignon <corentin.rossignon@gmail.com>
Bugzilla: https://bugs.freedesktop.org/104405
Reviewed-by: Adam Jackson <ajax@redhat.com>
(cherry picked from commit dbfbe58b94ec07a45fd6170f96bffec55051724e)
---
 glamor/glamor_xv.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/glamor/glamor_xv.c b/glamor/glamor_xv.c
index 31320d1..e0375d7 100644
--- a/glamor/glamor_xv.c
+++ b/glamor/glamor_xv.c
@@ -62,6 +62,8 @@ typedef struct tagREF_TRANSFORM {
 static const glamor_facet glamor_facet_xv_planar = {
     .name = "xv_planar",
 
+    .version = 120,
+
     .source_name = "v_texcoord0",
     .vs_vars = ("attribute vec2 position;\n"
                 "attribute vec2 v_texcoord0;\n"
-- 
2.8.3

